analytics-integrations/moengage/src/main/res/layout/moe_image_message_button.xml

```diff
@@ -0,0 +1,63 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We won't be shipping these in our SDK, so I'd recommend using the AAR instead of the JAR.